Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
     triggers { cron(cron_string) }
     environment {
         SERVICE = 'ocpmetal/assisted-service'
-        ISO_CREATION = 'installer-image-build'
+        ISO_CREATION = 'assisted-iso-create'
```

Comment:
This build will not be tested in subsystem because the policy is pull Always. 
in order to test the changes you will need to set the job image pull policy to Never, so it should be configured with environment variables.

Reply:
iso creation image is changed very rarely. So, i suggest we will not change it now , but i have a ticket opened MGMT-1838, and i will implement it after the release